features/networking/sdn.feature

```diff
@@ -224,7 +224,7 @@ Feature: SDN related networking scenarios
       | bash | -c | pgrep ovs-vswitchd |
     Then the step should succeed
     When I run command on the "<%= cb.node_name %>" node's sdn pod:
-      | bash | -c | pgrep ovs-vswitchd \| xargs kill -9 |
+      | bash | -c | pgrep ovs-vswitchd \| xargs kill |
```

Comment:
could we use `pkill`?    `pkill ovs-vswitchd`

Reply:
Thanks @rbbratta, updated it!